infrastructure/demo/main/ecr.tf

```diff
@@ -0,0 +1,5 @@
+resource "aws_ecr_repository" "web_ecr_repo" {
+  name         = var.ecr_repo_name
+  force_delete = true
```

Comment:
Also getting red underline by Terraform that `Unexpected attribute: An attribute named "force_delete" is not expected here` but resource creation worked fine..

Reply:
The variable is there but is optional according to the [documentation](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecr_repository)